terraform/templates/hab-sup.service

```diff
@@ -2,6 +2,7 @@
 Description=Habitat Supervisor
 
 [Service]
+Environment="HAB_FEAT_EVENT_STREAM=1"
```

Comment:
NOTE: Something I found in testing is that if your flags *don't* include the streaming options, including this environment variable will hard fail. Should we parameterize?

Reply:
that's a good idea, so that'd come into place if folks are doing national-parks demo without Automate.

Reply:
Correct. I *think* it still works if you end up pointing the event flags at something invalid, but if you define the environment var and neglect any of the `--event-FOO` flags in your args, it'll fail to launch the supervisor. Some sort of logic to the logical effect of "only do this if flags x, y, and z are also set" would be super neat, but that's more complicated, and a simple toggle param is probably the path of least resistance.